### PR TITLE
Improving app

### PR DESCRIPTION
**Pull request by tjmelo**
Proposal
---
- Improving README.md
- Some adjusts in CI

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,11 +15,9 @@ WORKDIR /${TMX_PATH}
 
 COPY package*.json ./
 
-RUN npm ci --only=prodution
+RUN npm ci --only=production --ignore-scripts
```
Comment (tm-wd) [inline Dockerfile:18]: Has `ignore-scripts` some impact on the analysis process of the build?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
 ![Passing](https://img.shields.io/github/actions/workflow/status/tjmelo/tmx-mercury/github-actions-demo.yml)
-![Version](https://img.shields.io/github/v/release/tjmelo/tmx-mercury)
+
+## Table of content
+
+-   [About app](#about-app)
+-   [Preview Screens](#preview-screens)
+-   [Project details](#project-details)
+    -   [Simple Develop Process](#simple-develop-process)
+    -   [Docker develop Process](#docker-develop-process)
```
Comment (tm-wd) [inline README.md:14]:
```suggestion
    -   [Docker Develop Process](#docker-develop-process)
```

File: Dockerfile (hunk 1 above)
Comment (tjmelo) [reply Dockerfile:18]: No, this script prevents malicious command execution in CI. (prestart)

Comment (tm-wd) [reply Dockerfile:18]: Ok! :+1:

**Pull request merged**